src/lib/utils/fonts.ts

```diff
@@ -60,16 +59,52 @@ const FONT_CONFIG: Record<
   },
 };
 
+const CYRILLIC_FONT_CONFIG: Record<Font, FontSettings> = {
+  Default: {
+    fontFamily: "Basis33",
+    xxs: [20, 14],
+    xs: [24, 14],
+    sm: [30, 20],
+    base: [36, 26],
+    lg: [42, 32],
+  },
+  "Sans Serif": {
+    fontFamily: "sans-serif",
+    xxs: [14, 14],
+    xs: [16, 16],
+    sm: [18, 18],
+    base: [25, 25],
+    lg: [30, 30],
+  },
+  Bold: {
+    fontFamily: "Born2bSporty",
+    xxs: [18, 12],
+    xs: [24, 14],
+    sm: [30, 20],
+    base: [36, 26],
+    lg: [42, 32],
+  },
+  "Chunky (Old)": {
+    fontFamily: "Russo One",
+    xxs: [14, 14],
+    xs: [16, 16],
+    sm: [18, 18],
+    base: [25, 25],
+    lg: [30, 30],
+  },
+};
+
 export function initialiseFont() {
   const font = getCachedFont();
 
-  if (font !== "Default") {
-    changeFont(font);
-  }
+  changeFont(font);
 }
 
 export function changeFont(font: Font) {
-  const config = FONT_CONFIG[font];
+  const lang = localStorage.getItem("language") || "en";
+  const config = ["ru"].includes(lang)
```

Comment:
Picking correct config depending on language(currently we have only Russian language which uses Cyrillic fonts, but that can change in future if we add Ukrainian and other languages)